app/views/finders/show_all_content_finder.html.erb

```diff
@@ -40,6 +40,10 @@
           } %>
         </div>
 
+        <div class="spelling-suggestions">
```

Comment:
`js-spelling-suggestions` class isn't needed here? I'm not able to check if the JS is being picked up regardless.

Reply:
Good question: the `js-spelling-suggestions` ID is used by the existing finder UI to do its wonky JS-based page refresh.

As we will not be doing this for the new UI, at least not initially, any change of keywords/filters will trigger a full page refresh and not need any JS hacks to update parts of the page.